README.md

```diff
@@ -344,6 +344,7 @@ boards:
       devpath: the UDEV devpath to this uart for UART without serial number
       interfacenum:	(optional) The interfacenumber of the serial. (Used with two serial in one device)
       use_ser2net: 	True/False (Deprecated, ser2net is the default uart handler)
+      worker:          (optional) an host/IP where ser2net is running
```

Comment:
Would this be better as `ser2net_worker` as it's specifically for ser2net, like the other variables below?

Obviously `lavalab-gen.py` would need updating accordingly.

Reply:
It could be used for other purpose, with labgrid ser2net will be not used, but this information remains the same.

Reply:
Ah okay :)